src/redux/ui/validators/actions.ts

```diff
@@ -0,0 +1,40 @@
+import { Dispatch } from 'react';
+import { IReduxState } from '../../state';
+import { getChainId } from '../../preferences/selectors';
+import { PosBasicActionType } from '../../../core/blockchain/types/token';
+import { IAccountState } from '../../wallets/state';
+
+// actions consts
+export const GET_VALIDATORS = 'GET_VALIDATORS';
+export const ADD_VALIDATORS = 'ADD_VALIDATORS';
+
+export const fetchValidators = (account: IAccountState, posAction: PosBasicActionType) => async (
+    dispatch: Dispatch<any>,
+    getState: () => IReduxState
+) => {
+    const state = getState();
+    const blockchain = account.blockchain;
+    const chainId = getChainId(state, blockchain).toString();
+
+    const result = await fetch('http://127.0.0.1:8080/wallet-ui/validators/list', {
```

Comment:
You can move this code, but after you will make a rebase from my PR. A have created an ApiClient

Also use HttpClient with post